File: Service/Implement/DanhMucDangKyCapGiayService.cs
namespace Service.Implement
{
    public class DanhMucDangKyCapGiayService : BaseService<DanhMucDangKyCapGiay, IDanhMucDangKyCapGiayRepository>
    , IDanhMucDangKyCapGiayService
    {
    private readonly IDanhMucDangKyCapGiayRepository _DanhMucDangKyCapGiayRepository;
    public DanhMucDangKyCapGiayService(IDanhMucDangKyCapGiayRepository DanhMucDangKyCapGiayRepository) : base(DanhMucDangKyCapGiayRepository)
    {
    _DanhMucDangKyCapGiayRepository = DanhMucDangKyCapGiayRepository;
    }
    }
    }

File: CMS/src/app/shared/DanhMucDangKyCapGiay.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucDangKyCapGiay } from 'src/app/shared/DanhMucDangKyCapGiay.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucDangKyCapGiayService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucDangKyCapGiay";
    }
}

